Xamarin.Forms.Core/Shell/ShellContent.cs

```diff
@@ -12,6 +12,10 @@
 
 namespace Xamarin.Forms
 {
+	public class TabContent : ShellContent
+	{
+	}
+
 	[ContentProperty("Content")]
 	public class ShellContent : BaseShellItem, IShellContentController
```

Comment:
Could we have a Content alias, removing the `Shell` here as well?

Reply:
I would not, as ShellContent has a property Content. I'm fine with an alias, be we need a better name, something like "Page", but not that

Reply:
we'll probably merge as is for pre8 testing

But would there be a way to achieve what David is asking with simplifying down the names so we could have

```
<Shell>
        <Tab>
            <Content>
                
            </Content>
        </Tab>
</Shell>
```

it does seem confusing to now have something called
*TabContent* inside the Xamarin.Forms namespace that only belongs to Shell